src/managers/KeybindManager.cpp

```diff
@@ -1048,14 +1048,20 @@ SWorkspaceIDName getWorkspaceToChangeFromArgs(std::string args, PHLWORKSPACE PCU
         return {WORKSPACE_NOT_CHANGED, ""};
     }
 
-    const auto ID = PCURRENTWORKSPACE->m_iID;
-    if (const auto PWORKSPACETOCHANGETO = g_pCompositor->getWorkspaceByID(PPREVWS.id); PWORKSPACETOCHANGETO) {
-        if (PER_MON && PCURRENTWORKSPACE->m_iMonitorID != PWORKSPACETOCHANGETO->m_iMonitorID)
-            return {WORKSPACE_NOT_CHANGED, ""};
+    const auto ID                   = PCURRENTWORKSPACE->m_iID;
```

Comment:
uhh unintentional?

Reply:
the heck where did that come from, very much unintentional. must be some rebase/force-push mistake. sorry

Reply:
yeah its the third commit, latest on upstream/main that got added as an commit somehow inbetween my rebase force push to make CI format happy. will revert <3

Reply:
solved, short story my local branch didnt have https://github.com/hyprwm/Hyprland/commit/3fd6f981ed85c5e458a1a73f7c010ee02a8ff564 and inbetween rebase/force push it got added again here then. but reverted :)